doc/profiling.md

```diff
@@ -94,6 +98,22 @@ with the `LabeledNode(.node=child, .scc=childscc)` literal.
   451,529 and 372,446 records respectively (the numbered variables, e.g., `_0`)
   indicate one or more fields used to index the relation by.
 
+  In addition, if change profiling was enabled for some parts of the execution,
```

Comment:
should this section have a little snippet of a change profile?

Reply:
ack